packages/tools/bundleDefaultPlugins.ts

```diff
@@ -32,10 +32,13 @@ export const localPluginsVersion = async (defaultPluginDir: string, defaultPlugi
 
 async function downloadFile(url: string, outputPath: string) {
 	const response = await fetch(url);
+	if (!response.ok) {
+		throw new Error(`Cannot download file from ${url}`);
```

Comment:
Here too you might want to provide more info such as response.status or response.text

Reply:
Done :+1: